macaroon.go

```diff
@@ -232,6 +232,27 @@ func (m *Macaroon) Verify(rootKey []byte, check func(caveat string) error, disch
 	return nil
 }
 
+// VerifySignature verifies the signature of the given macaroon with respect
+// to the root key, but it does not validate any first-party caveats. Instead
+// it returns all the applicable first party caveats on success.
+//
+// The caller is responsible for checking them.
```

Comment:
.. for checking the returned first party caveats.

You can't seriously expect me to read all the paragraphs.